tests/cpydiff/module_array_constructor.py

```diff
@@ -0,0 +1,11 @@
+"""
+categories: Modules,array
+description: overflow checking is not implemented
+cause: micropython implements implicit truncation in order to reduce code size and execution time
+workaround: mask value
+"""
+import array
+
+value = 257
+a = array.array("b", [value & 0xff])
```

Comment:
the idea is for this test to produce different results on CPython vs MicroPython, so you need to remove the `& 0xff`

Reply:
OK, done.